lisa/wa.py

```diff
@@ -645,3 +645,38 @@ def get_stats(self, **kwargs):
             agg_cols=['iteration', 'frame_id'],
             **kwargs,
         )
+
+class WASysfsExtractorCollector(WAArtifactCollectorBase):
+    NAME = 'sysfs-extractor'
+
+    def __init__(self, wa_output, path, type='diff', df_postprocess=None):
```

Comment:
Unless `df_postprocess` is directly used here, it's probably best to just use `**kwargs`